src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1223,7 +1223,9 @@ class ValidationCount
 {
 public:
     int trustedValidations, nodesUsing;
-    NodeID highNodeUsing, highValidation;
+    // Ledger hashes to break ties
+    uint256 highUsingHash, highTrustedValHash;
```

Comment:
Since this class is only used in `checkLastClosedLedger` in a `hash_map<uint256, ValidationCount>`, I'd consider removing both `highUsingHash` and `highTrustedValHash` (we already have the ledger hash as the key). Then you could handle tiebreaking outside of the comparison operator.